.all-contributorsrc

```diff
@@ -266,6 +266,13 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "natecavanaugh",
+      "name": "Nate Cavanaugh",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/116871?v=3",
+      "profile": "http://alterform.com",
+      "contributions": []
```

Comment:
Could you add `"code"` to this array and then run `npm start generateContributors`?